server/test/src/main/java/org/eclipse/lsp/cobol/test/engine/UseCaseEngine.java

```diff
@@ -278,6 +280,7 @@ public AnalysisResult runTest(
                 .build(),
             languageId);
     assertResultEquals(actual, document.getTestData());
+    UseCaseUtils.storeDocumentToUnitTextExtensionContext(document.getText(), document.getCopybooks(), document.getTestData());
```

Comment:
Do I understand correctly that the preprocessed code will be stored in the output?

Reply:
Yes